src/vm/typedesc.cpp

```diff
@@ -837,6 +837,12 @@ OBJECTREF ParamTypeDesc::GetManagedClassObject()
             pLoaderAllocator->ClearHandle(hExposedClassObject);
         }
 
+        if (OwnsTemplateMethodTable())
+        {
+            // Set the handle on template methodtable as well to make Object.GetType for arrays take the fast path
```

Comment:
LGTM. For my education, where does the Fast path for Array consuming this field?